.openshift-ci/build/Dockerfile.build_root

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/stackrox-io/apollo-ci:collector-0.3.42
+
+# oc
+RUN sudo wget --no-verbose -O oc.tgz https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz
+RUN sudo tar -xf oc.tgz
+RUN sudo install openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin
+RUN sudo rm -rf openshift-* oc.tgz
+RUN command -v oc
```

Comment:
Does this image exist only for `oc`? There seems to be a way to inject existing `oc` into the steps [[1]], would it be easier?

[1]: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli